third_party/terraform/utils/node_config.go.erb

```diff
@@ -502,4 +509,58 @@ func flattenSandboxConfig(c *containerBeta.SandboxConfig) []map[string]interface
 	}
 	return result
 }
+
+func containerNodePoolLabelsSuppress(k, old, new string, d *schema.ResourceData) bool {
+	o, n := d.GetChange("node_config.0.sandbox_config.0.sandbox_type")
```

Comment:
I believe this is trying to suggest that we only care about the sandbox label if sandbox_type is set, is that correct? If so can you add a comment to that regard?

Reply:
This is indeed the case; I added a comment about this to both methods.